source/configuration.txt

```diff
@@ -0,0 +1,22 @@
+.. _mongoid-configuration:
+
+=============
+Configuration
+=============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: ruby framework, odm, setup, config
+
+.. toctree::
+   :caption: Configuration
+
+   Sharding </configuration/sharding>
+
+In this section, you can learn how to configure different options with {+odm+}.
```

Comment:
This page will eventually have more than just sharding config options. Eventually it will have things like "Collection Configuration" and "Persistence Configuration" and some others